predator/templates/configmap.yaml

```
@@ -37,7 +37,10 @@ data:
   STREAMING_PLATFORM_HEALTH_CHECK_TIMEOUT_MS: {{ .Values.streaming.platformHealthCheckTimeoutMs | quote }}
 {{ end }}
 {{ if .Values.streaming.excludedAttributes }}
-STREAMING_EXCLUDED_ATTRIBUTES: {{ .Values.streaming.excludedAttributes }}
+  STREAMING_EXCLUDED_ATTRIBUTES: {{ .Values.streaming.excludedAttributes }}
+{{ end }}
+{{ if .Values.max_upload_file_size_mb }}
+  MAX_UPLOAD_FILE_SIZE_MB: {{ .Values.max_upload_file_size_mb }}
```

```suggestion
  MAX_UPLOAD_FILE_SIZE_MB: {{ .Values. maxUploadFileSizeMB | quote }}
```

Since this is a number we need to add quotes